spec-kit/memory/constitution.md

```diff
@@ -0,0 +1,69 @@
+# Stock Indicators for .NET Constitution
+
+## Core Principles
+
+### I. Mathematical Precision (NON-NEGOTIABLE)
```

Comment:
This isn’t entirely true, soften, based on actual prevalence of the current codebase